Patch-Source: https://sources.debian.org/data/main/libt/libtest-unit-perl/0.25-7/debian/patches/20_fix-defined-array-warnings.patch

Description: Fix defined(@array) deprecation warning
 Also fixes test suite failures where STDERR was checked
Author: Axel Beckert <abe@debian.org>
Bug: https://rt.cpan.org/Public/Bug/Display.html?id=102732

Index: libtest-unit-perl/lib/Test/Unit/TestCase.pm
===================================================================
--- libtest-unit-perl.orig/lib/Test/Unit/TestCase.pm	2013-12-25 21:59:45.182407866 +0100
+++ libtest-unit-perl/lib/Test/Unit/TestCase.pm	2013-12-25 22:08:27.777044278 +0100
@@ -103,7 +103,7 @@
     my $class = ref($_[0]) || $_[0];
     my @tests = ();
     no strict 'refs';
-    if (defined(@{"$class\::TESTS"})) {
+    if (@{"$class\::TESTS"}) {
         push @tests, @{"$class\::TESTS"};
     }
     else {
